### PR TITLE
Add h2load client app

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

File: src/H2LoadClient/Program.cs [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+            app.HelpOption("-h|--help");
+            var optionUrl = app.Option("-u|--url <URL>", "The server url to request", CommandOptionType.SingleValue);
+            var optionConnections = app.Option<int>("-c|--connections <N>", "Total number of connections to use", CommandOptionType.SingleValue);
+            var optionThreads = app.Option<int>("-t|--threads <N>", "The number of native threads", CommandOptionType.SingleValue);
+            var optionStreams = app.Option<int>("-m|--streams <N>", "Max concurrent streams to issue per session", CommandOptionType.SingleValue);
+            var optionTimeout = app.Option<int>("-T|--timeout <N>", "Timeout to keep the connection open", CommandOptionType.SingleValue);
+            var optionWarmup = app.Option<int>("-w|--warmup <N>", "Duration of the warmup in seconds", CommandOptionType.SingleValue);
+            var optionDuration = app.Option<int>("-d|--duration <N>", "Duration of the test in seconds", CommandOptionType.SingleValue);
+            var optionProtocol = app.Option<string>("-p|--protocol <S>", "The SignalR protocol to use 'json', 'messagepack'", CommandOptionType.SingleValue);
```
Comment (sebastienros) [inline src/H2LoadClient/Program.cs:39]: Fix description?

Comment (sebastienros) [reply src/H2LoadClient/Program.cs:39]: h2, h2c, ...

File: build/benchmarks.yml [hunk 2]
```diff
@@ -120,89 +24,3 @@ jobs:
       warmup: 5
       duration: 15
       connection: 'ASP.NET Benchmarks Queue'
-
-- job: SignalR
```
Comment (halter73) [inline build/benchmarks.yml:124]: Any particular reason for removing the SignalR job from benchmarks.yml as part of this PR?

Comment (JamesNK) [reply build/benchmarks.yml:124]: temp debugging

**Pull request merged**